lib/puppet/util/instance_loader.rb

```diff
@@ -41,7 +41,7 @@ def instance_docs(type)
 
     # Use this method so they all get loaded
     loaded_instances(type).sort { |a,b| a.to_s <=> b.to_s }.each do |name|
-      mod = self.loaded_instance(name)
+      mod = self.loaded_instance(type, name)
```

Comment:
this is terrifying

Reply:
Fortunately only used for documentation generation.